pkg/resourcemanager/iam/authorizers.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/Azure/go-autorest/autorest/azure/auth"
 )
 
+// TODO: This file was adopted from another file... we should note that
+// TODO: looks like here? https://github.com/Azure-Samples/azure-sdk-for-go-samples/blob/master/internal/iam/authorizers.go
+
+// TODO: No we need to actually delete this file and all of its usage I think...
+// TODO: just use https://github.com/Azure/go-autorest/blob/master/autorest/azure/auth/auth.go NewAuthorizerFromEnvironment
```

Comment:
Should these TODOs be addressed now?

Reply:
They are actually wrong, will fix and remove TODO as what remains is mostly "notes" rather than a TODO.